docker/Dockerfile

```diff
@@ -29,16 +28,18 @@ RUN mkdir -p ~/catkin_ws/src
 # Set the working directory
 WORKDIR .
 
-ENV CMAKE_CXX_COMPILER /usr/lib/llvm-4.0/bin/clang++
+ENV CMAKE_CXX_COMPILER /usr/lib/llvm-10/bin/clang++
 
-RUN source /opt/ros/melodic/setup.bash;\
+RUN source /opt/ros/$ROS_DISTRO/setup.bash;\
```

Comment:
should we define the $ROS_DISTRO as a global variable for the Docker configuration? then it can also be used for the line 2.

Reply:
in that case we need to take care what python version is being used

Reply:
yes, sorry. I added this comment before I tested the changes. It is not realistic to have a single docker file for all the alternatives. I will close the discussion as resolved

Reply:
I will push another commit I am currently working on. Created 2 folders inside `docker` folder.